root/release/edit/tracklist.tt

```diff
@@ -1,7 +1,21 @@
 <script type="text/html" id="template.track-parser">
   [%# the track-parser-dialog is a simplified version of the add-medium-dialog. %]
-  <p>[% l('Enter a tracklist below:') %]</p>
-
+  <p>[% l('Enter a tracklist below:') %] (<a href="#" data-bind="click: $root.trackParser.toggleParserHelp">[% l('help') %]</a>)</p>
+  <div data-bind="visible: $root.trackParser.parserHelpVisible">
+    <p>[% l('If you have a tracklist you can copy and paste, you can input it here.') %]</p>
+    <p>[% l('By default, the parser expect the following format:') %]</p>
+    <ul>
+      <li>1. Love Me Do - The Beatles (2:23)</li>
+      <li>2. P.S. I Love You - The Beatles (2:03)</li>
+    </ul>
+    <p>
+      [% l('You can use the checkboxes under the parser to deactivate some sections.') %]
+      [% l('For example, you can deactivate "Lines contain track artists" if your data has no track artists, or "Use track lengths" if you have track durations but want the parser to ignore them.') %]
```

Comment:
Would it make sense to combine these strings?

Reply:
I guess so :) It was mostly for readability but since they go together it's probably easier for translators to have them as one.